app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/EditServiceMonitoringTest.php

```diff
@@ -133,10 +133,6 @@
             ServiceMonitoringTargetRequestFactory::new()->state(['domain' => str()->random(256)]),
             ['domain' => 'max'],
         ],
-        'domain url' => [
-            ServiceMonitoringTargetRequestFactory::new()->state(['domain' => 'invalid-url']),
-            ['domain' => 'url'],
-        ],
```

Comment:
If we are going to remove this here then we need the same test as you have in the Create test.